internal/pkg/policy/self.go

```diff
@@ -174,9 +183,35 @@ func (m *selfMonitorT) processPolicies(ctx context.Context, policies []model.Pol
 		policy := latest[i]
 		if m.policyID != "" && policy.PolicyID == m.policyID {
 			m.policy = &policy
+			esOut, err := m.getPolicyOutput()
+			if err != nil {
+				if !errors.Is(err, ErrNoPolicyUpdate) {
+					m.log.Warn().Err(err).Str(logger.PolicyID, m.policyID).Msg("Failed to get fleet-server output")
```

Comment:
`updateState` would get called directly after this log line, and as that's implemented now it would immediately change the state back to healthy and i'm hesitant to add more state-tracking within the self-monitor (especially for parsing errors)